Gemfile.lock

```diff
@@ -209,8 +209,8 @@ GEM
       unicode-display_width (>= 1.4.0, < 3.0)
     rubocop-ast (1.4.1)
       parser (>= 2.7.1.5)
-    rubocop-shopify (1.0.7)
-      rubocop (~> 1.4)
+    rubocop-shopify (2.0.1)
+      rubocop (~> 1.11)
```

Comment:
More context: https://github.com/Shopify/ruby-style-guide/pull/238

This change enforces that all files use double quotes. The PR above suggests that this is the preference for Rails 6.1 as well. 

Making this change in this PR would mean **a lot** of files are changed, and not doing it would mean our rubocop build fails. How does everyone feel about making this change in this PR?

Reply:
For reference, running `bundle exec rubocop -a` makes 1200+ corrections.

Reply:
Alternatively, is it alright to just not jump to the latest `rubocop-shopify` release?

Reply:
I would keep this one as is and set up a separate PR containing only the auto-updates.